menas/ui/components/run/runDetail.view.xml

```diff
@@ -32,7 +32,7 @@
             </Bar>
         </customHeader>
         <content>
-            <IconTabBar id="runIconTabBar" select="tabSelect">
+            <IconTabBar id="runIconTabBar" select="tabSelect" stretchContentHeight="true" expandable="false">
```

Comment:
Yeah that was the property I found about just before you mentioned you would take over. 😉